spec/features/users/edit_profile_spec.rb

```diff
@@ -156,5 +156,17 @@
       expect(page).to have_content('33333')
       expect(page).to have_content('merchant@gmail.com')
     end
+
+    it "I can't change my email address to one belonging to another user" do
+      allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(@user)
+
+      visit '/profile/edit'
```

Comment:
```suggestion
      visit '/profile/edit/profile'
```
Based on the last pull request, we need this route to be updated to the above.